instrumentation/ruby_kafka/test/opentelemetry/instrumentation/ruby-kafka/events/connection/request_test.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
```

Comment:
```suggestion
# Copyright The OpenTelemetry Authors
```
Can we fix copyright notices throughout this PR, or do you plan to do that with #512?

Reply:
I'll fix up any leftover copy pasted cruft I can find here.